.gitlab-ci.yml

```diff
@@ -0,0 +1,118 @@
+      cd build &&
+      cmake -DCMAKE_INSTALL_PREFIX=${CI_PROJECT_DIR}/kokkos.install -DCMAKE_CXX_COMPILER=${CI_PROJECT_DIR}/kokkos/bin/nvcc_wrapper -DCMAKE_CXX_EXTENSIONS=OFF -DKokkos_ENABLE_SERIAL=ON -DKokkos_ENABLE_OPENMP=ON -DKokkos_ENABLE_CUDA=ON -DKokkos_ENABLE_CUDA_LAMBDA=ON -DKokkos_ARCH_POWER9=ON -DKokkos_ARCH_VOLTA70=ON .. &&
+      make &&
+      make install
+  tags:
+    - nobatch
+  artifacts:
+    paths:
+      - kokkos.install/
+
+BuildArborX:
+  stage: buildArborX
+  script:
+    - module load gcc/7.4.0 cuda cmake git
+    - mkdir build && cd build &&
+      cmake -DCMAKE_PREFIX_PATH="${CI_PROJECT_DIR}/kokkos.install;${CI_PROJECT_DIR}/benchmark.install;${CI_PROJECT_DIR}/boost.install" -DARBORX_ENABLE_MPI=ON -DARBORX_ENABLE_TESTS=OFF -DARBORX_ENABLE_BENCHMARKS=ON -DCMAKE_CXX_COMPILER=${CI_PROJECT_DIR}/kokkos.install/bin/nvcc_wrapper .. &&
+      make VERBOSE=1 ArborX_BoundingVolumeHierarchyRegression.exe
+    - export OMP_PROC_BIND=spread 
+    - export OMP_PLACES=threads
+    - cp ./benchmarks/bvh_driver/ArborX_BoundingVolumeHierarchyRegression.exe /ccsopen/proj/csc333/ArborX_BoundingVolumeHierarchyRegressionBranch.exe
+  tags:
+    - nobatch
+
+RunBranch:
+  stage: runBenchmarkBranch
+  script:
+    - module load gcc/7.4.0 cuda cmake git
+    - jsrun --smpiargs="-gpu" --nrs 1 --gpu_per_rs 6 --rs_per_host 1 --np 1 --latency_priority CPU-CPU --launch_distribution packed --bind packed:1 /ccsopen/proj/csc333/ArborX_BoundingVolumeHierarchyRegressionBranch.exe --benchmark_format=json --no-header --benchmark_repetitions=20 | tee /ccsopen/proj/csc333/arborx-branch.json
+  tags:
+    - batch
+
+BuildArborXMaster:
+  stage: buildArborX
+  script:
+    - module load gcc/7.4.0 cuda cmake git
+    - git fetch
+    - git worktree remove ${CI_PROJECT_DIR}/arborx-master
+    - git worktree add ${CI_PROJECT_DIR}/arborx-master origin/master-ci 
+    - cd ${CI_PROJECT_DIR}/arborx-master
+    - mkdir build_master && cd build_master &&
+      cmake -DCMAKE_PREFIX_PATH="${CI_PROJECT_DIR}/kokkos.install;${CI_PROJECT_DIR}/benchmark.install;${CI_PROJECT_DIR}/boost.install" -DARBORX_ENABLE_MPI=ON -DARBORX_ENABLE_TESTS=OFF -DARBORX_ENABLE_BENCHMARKS=ON -DCMAKE_CXX_COMPILER=${CI_PROJECT_DIR}/kokkos.install/bin/nvcc_wrapper .. &&
+      make VERBOSE=1 ArborX_BoundingVolumeHierarchyRegression.exe
+    - export OMP_PROC_BIND=spread 
+    - export OMP_PLACES=threads
+    - cp ./benchmarks/bvh_driver/ArborX_BoundingVolumeHierarchyRegression.exe /ccsopen/proj/csc333/ArborX_BoundingVolumeHierarchyRegressionMaster.exe
+  tags:
+    - nobatch
+
+RunMaster:
+  stage: runBenchmarkMaster
+  script:
+    - module load gcc/7.4.0 cuda cmake git
+    - jsrun --smpiargs="-gpu" --nrs 1 --gpu_per_rs 6 --rs_per_host 1 --np 1 --latency_priority CPU-CPU --launch_distribution packed --bind packed:1 /ccsopen/proj/csc333/ArborX_BoundingVolumeHierarchyRegressionMaster.exe --benchmark_format=json --no-header --benchmark_repetitions=20 | tee /ccsopen/proj/csc333/arborx-master.json
```

Comment:
Don't think we need 20 repetitions. Can we also unify all these arguments here with the ones for branch, so that we don't accidentally change one without changing the other?